sdv/model.py

```diff
@@ -841,11 +667,159 @@ async def get(self) -> TypedDataPointResult[List[str]]:
             raise
 
     async def set(self, value: List[str]):
+        await self._set(value, self.__class__.__name__)
+
+    def create_broker_data_point(self, value: List[str]):
+        array = StringArray(values=value)
+        return BrokerDatapoint(string_array=array)
+
+
+class BatchSetBuilder:
+    """Collects data points to be updated in a single 'atomic' set"""
+
+    def __init__(self, client):
+        self.__client = client
+        self.__nodes = {}
+
+    def preset(self, node: DataPoint, value) -> "BatchSetBuilder":
```

Comment:
Not a big fan of calling this `preset`. A preset is a pre-configured value for me which differs from what we are doing here.

Reply:
I put `preset` here as a suggestion. My thinking is that `update` is somehow suggesting that it is actually doing the update and `preset` is name-wise closer to `set`. But maybe `preselect` would be close to what I wanted to express ..

Reply:
What about 
```python
self.Vehicle.setMany() # or setAtomic() to express what this is used for
  .add(Vehicle.Speed, 10.0)
  .add(Vehicle.Seat.Row1.Pos1.Position, 80)
  .apply()
```

This will also solve your problem of having to add @overrides in 2 places

Reply:
I also agreed to have a simple **update** instead, I share the option above on the preset.
- I would suggest to go for **update** and simple extend the documentation to explain how to use the builder.

Reply:
Done